{"name": "test_zhs_seleniumlogintest", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable", "trace": "self = <testcases.user.test_zhs_seleniumlogintest.Test_seleniumlogintest object at 0x000002A1625E1C10>\n\n    def test_zhs_seleniumlogintest(self):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        chromedriver_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))),\n                                         \"config\\chromedriver.exe\")\n        logger.info(chromedriver_path)\n        # browser = webdriver.Chrome(chromedriver_path)\n        firefoxdriver_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))),\n                                          \"config\\cgeckodriver.exe\")\n>       browser = webdriver.firefox(firefoxdriver_path)\nE       TypeError: 'module' object is not callable\n\ntest_zhs_seleniumlogintest.py:27: TypeError"}, "attachments": [{"name": "log", "source": "8f08398f-64d2-465b-8bc4-67fbcd34cf1a-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "a610809d-8271-492c-b882-b427d005d9d3-attachment.txt", "type": "text/plain"}], "start": 1624429516045, "stop": 1624429516045, "uuid": "ea932f76-1b10-4f9c-9dd5-a5a157528a5f", "historyId": "c06aec85e63e41e2b940ef2c08025c58", "testCaseId": "e6ed18b5be4e055550dc72885e918ffe", "fullName": "testcases.user.test_zhs_seleniumlogintest.Test_seleniumlogintest#test_zhs_seleniumlogintest", "labels": [{"name": "parentSuite", "value": "testcases.user"}, {"name": "suite", "value": "test_zhs_seleniumlogintest"}, {"name": "subSuite", "value": "Test_seleniumlogintest"}, {"name": "host", "value": "DESKTOP-FRVNKTB"}, {"name": "thread", "value": "3984-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.user.test_zhs_seleniumlogintest"}]}